src/sentry/web/urls.py

```diff
@@ -131,6 +132,8 @@ def init_all_applications():
             debug_auth_views.DebugAuthConfirmIdentity.as_view()),
         url(r'^debug/auth-confirm-link/$',
             debug_auth_views.DebugAuthConfirmLink.as_view()),
+        url(r'^debug/icons/$',
+            DebugIconsView.as_view()),
```

Comment:
This could just be `TemplateView.as_view(template_name='sentry/debug/icons.html')` and avoid the `DebugIconsView` class entirely.